libs/SmartSync/SmartSync/Classes/Model/SFObject.h

```diff
@@ -27,6 +27,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+NS_SWIFT_NAME(SObject)
```

Comment:
What should the name be?

Reply:
Maybe `SalesforceObject`? `SObject` is fine too.